backend/src/tests/demo_db/conftest.py

```diff
@@ -0,0 +1,159 @@
+import asyncio
+import time
+from contextlib import asynccontextmanager
+from typing import AsyncIterator
+
+import httpx
+import mirakuru
+import pytest
+from app.main import app
+from app.models import BaseDAO, UserDAO
+from sqlalchemy import create_engine
+from sqlalchemy.ext.asyncio import AsyncSession, create_async_engine
+from sqlalchemy.orm import sessionmaker
+
+# shell command to launch ephemeral *in-memory* cockroach db
```

Comment:
$1 says low hanging fruit could figure out a cmdline to get the in-mem only driver, no verbose logging, and no automatic entry into the sql shell, but this suggestion from CRL dude is awesome.

Reply:
yep definitely want to give this repo back to cockroach labs and ask what they'd change or streamline.  Maybe a new `cockroach testdb` command will come out of it all?